src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/ISyntaxFactsService.cs

```diff
@@ -262,7 +264,9 @@ internal interface ISyntaxFactsService : ILanguageService
         bool IsDirective(SyntaxNode node);
         bool IsForEachStatement(SyntaxNode node);
         bool IsLockStatement(SyntaxNode node);
-        bool IsUsingStatement(SyntaxNode node);
+#nullable enable
+        bool IsUsingStatement(SyntaxNode? node);
+#nullable disable
```

Comment:
These should all be #nullable restore -- when we flip it on for the project globally we don't want these staying disabled.